### DISCUSSION TITLE
第3节课笔记(25班)

### DISCUSSION THREAD

**Discussion by zcg-joker** [InternLM/Tutorial#340] [笔记]
笔记链接放在这里~
作业链接**不放在**这里！

**Comment by sxswz213** [comment]
https://blog.csdn.net/sxswz/article/details/135562461?spm=1001.2014.3001.5501

**Reply by zcg-joker** [reply]
很棒很棒~

**Comment by IdealDestructor** [comment]
https://blog.deqiang.wang/2024/01/15/Project-LLM3/

**Comment by BaiYu96** [comment]
https://github.com/BaiYu96/internLM-learm/blob/main/note/03-Langchain%2BInternLM.md

**Comment by fxlt** [comment]
https://github.com/fxlt/internLM-Study/blob/main/3.%E5%9F%BA%E4%BA%8E%20InternLM%20%E5%92%8C%20LangChain%20%E6%90%AD%E5%BB%BA%E4%BD%A0%E7%9A%84%E7%9F%A5%E8%AF%86%E5%BA%93.md

**Comment by zengya55** [comment]
https://github.com/zengya55/InterLM_Learning/blob/main/lesson3/lesson3%20%E6%90%AD%E5%BB%BA%E4%BD%A0%E7%9A%84%E7%9F%A5%E8%AF%86%E5%BA%93%E7%AC%94%E8%AE%B0.md

**Comment by juxingyiwan** [comment]
https://github.com/juxingyiwan/InternLM_Camp_202401/blob/main/Lecture3/Lecture3%E8%AF%BE%E7%A8%8B%E7%AC%94%E8%AE%B0.pdf

**Comment by LordHidy** [comment]
https://github.com/LordHidy/InternLM_learning/blob/main/Notes/Lesson_3.md

**Comment by MQDCY** [comment]
https://blog.csdn.net/weixin_48396757/article/details/135666016?spm=1001.2014.3001.5502

**Comment by ykuang** [comment]
https://github.com/ykuang/internlm-tutorial/blob/main/notes/lesson3.md

**Comment by Cyberworldvoyager** [comment]
https://github.com/Cyberworldvoyager/Interlm_lecture/blob/main/%E7%AC%AC%E4%B8%89%E8%8A%82%E8%AF%BE%E7%AC%94%E8%AE%B0.pdf

**Comment by liliang1918** [comment]
https://blog.csdn.net/lalala12ll/article/details/135609077

**Comment by FableFatale** [comment]
https://www.cnblogs.com/yunyetui/p/17968851

**Comment by wade361** [comment]
https://github.com/wade361/-Wade/issues/4

**Comment by GuoYiFantastic** [comment]
https://github.com/GuoYiFantastic/InternLM_training_camp/tree/main/%E5%9F%BA%E4%BA%8E%20InternLM%20%E5%92%8C%20LangChain%20%E6%90%AD%E5%BB%BA%E4%BD%A0%E7%9A%84%E7%9F%A5%E8%AF%86%E5%BA%93
前一段时间赶论文去了（但毫无成果lol），现在补一下缺的笔记和作业~

**Comment by juxingyiwan** [comment]
https://github.com/juxingyiwan/InternLM_Camp_202401/tree/main/Lecture3
